### PR TITLE
remove console.log

### PR DESCRIPTION
**Pull request by leggsimon**
@ironsidevsquincy

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
Might be useful to leave this in - it'll only appear in the playground. Can put

```
/* eslint no-console: 0 */
```

at the top of the file to allow it

**Comment by leggsimon** [comment]
To be honest I don't really see why it's useful? It only logs out the query that you've typed in...

**Comment by ironsidevsquincy** [comment]
After it's been encoded and posted, and the variables have been `JSON.parse`d, so was quite useful when building to see exactly what was being sent to graphql, but yeah, might not be useful now

File: server/lib/helpers/capify-metadata.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ const convertMetadata = (metadata, taxonomy) => metadata
 	.map(metadata => ({ term: { id: metadata.idV1 }}));
 
 /**
- * Take ESv3 style metadata and convert it back to CAPI style
- */
+	* Take ESv3 style metadata and convert it back to CAPI style
+	*/
```
Comment (ironsidevsquincy) [inline server/lib/helpers/capify-metadata.js]: weird spacing?

Comment (leggsimon) [inline server/lib/helpers/capify-metadata.js]: won't allow spaces :trollface:

Comment (ironsidevsquincy) [inline server/lib/helpers/capify-metadata.js]: install the latest `lintspaces-cli`

**Pull request merged**